app/services/return-url.js

```diff
@@ -55,13 +55,16 @@ define(['app', 'lodash'], function (app, _) { 'use strict';
         //============================================================
         function navigate(url) {
 
+            const isSaml  = url.includes('/saml/signin')
```

Comment:
Regex will be appropriate here, its be true if `https://accountc.cbd.int/login?return=/saml/signin`